src/Tizen.NUI/src/public/XamlBinding/BindableObject.cs

```diff
@@ -272,7 +272,7 @@ internal void SetValueAndForceSendChangeSignal(BindableProperty property, object
             if (property == null)
                 throw new ArgumentNullException(nameof(property));
 
-            if (true == isCreateByXaml)
+            if (true == IsBinded)
```

Comment:
If Some property setter was use `SetValueAndForceSendChangeSignal` and didn't use data binding, it does not raise PropertyChanged event, even though property was changed, is it intended behavior? if you intent this behavior, Could you tell me why?

Reply:
Could you tell me why do you add `SetValueAndForceSendChangeSignal` instead `SetValue` I may provide a solution to resolve your origin issue

Reply:
"If Some property setter was use SetValueAndForceSendChangeSignal and didn't use data binding, it does not raise PropertyChanged event, even though property was changed, is it intended behavior? if you intent this behavior, Could you tell me why?"

Reply:
The Text of TextEditor/TextField can be changed by key input, so that the object which binded to the Text of TextEditor/TextField will not be notified.

To fix this issue, we listen the TextChanged event in TextEditor/TextField, and notify the binded instance.

Reply:
>so that the object which binded to the Text of TextEditor/TextField will not be notified.

Why? I don't understand the reason, What is problem when notified was raised?
 Xamarin.Forms Entry.Text also can be changed by key input but it works same like others property

Reply:
Simply, what's issue with using SetValue instead of SetValueAndForceSendChangeSignal on Text of TextField?

Reply:
> > so that the object which binded to the Text of TextEditor/TextField will not be notified.
> 
> Why? I don't understand the reason, What is problem when notified was raised? Xamarin.Forms Entry.Text also can be changed by key input but it works same like others property

Because user can set focus on the TextEditor/TextField, and then change the Text by press remote-control or keyboard, in this situation, the set method of property Text will not be called, so the binded object will not be notified.

To fix this issue, we listen the TextChanged event, in the listener, we set the Text of TextEditor/TextField again.
```
        public override bool IsCreateByXaml
        {
            get
            {
                return base.IsCreateByXaml;
            }
            set
            {
                base.IsCreateByXaml = value;

                if (value == true)
                {
                    this.TextChanged += (obj, e) =>
                    {
                        this.Text = e.TextEditor.Text;  // <==== Set Text again.
                    };
                }
            }
        }
```

When the Text is reset, if call "SetValue", the original value of Text is same to new value, so that it will not notify the binded object
```
                if ((!same || raiseOnEqual))  //<==== orignal value is same to new value, this condition can't be matched.
                {
                    property.PropertyChanged?.Invoke(this, original, value);

                    if (binding != null && !currentlyApplying)
                    {
                        applying = true;
                        binding.Apply(true);
                        applying = false;
                    }

                    OnPropertyChanged(property.PropertyName);  // <==== Binded objects are notified here.
```

so we add the method SetValueAndForceSendChangeSignal, to force notify the binded object.

Reply:
Now I have removed the method SetValueAndForceSendChangeSignal and added a new method, to make the code more clearly.